src/components/reusable-ui/navbar/Navbar.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+import Logo from "../Logo";
+import {theme} from "../../../theme"
+import NavbarRightSide from './NavbarRightSide';
+import { refreshPage } from '../../../utils/window';
+
+export default function Navbar({username}) {
+  return (
+    <NavbarStyled>
+        <Logo className={"logo-order-page"} onClick={() => refreshPage()} />
```

Comment:
```suggestion
        <Logo className={"logo-order-page"} onClick={refreshPage} />
```

OPTIONNEL : tu peux simplifier à refreshPage (sans les parenthèse, ça fonctionne pareil et c'est plus court). C'est ma faute, j'ai vu que une fois après avoir tournée la vidéo